web/frontend/src/pages/admin/Admin.tsx

```diff
@@ -14,7 +14,7 @@ const Admin: FC = () => {
   const [users, setUsers] = useState([]);
   const [loading, setLoading] = useState(true);
 
-  const abortController = new AbortController();
+  const abortController = useMemo(() => new AbortController(), []);
```

Comment:
See https://stackoverflow.com/questions/69572613/how-to-wrap-the-initialisation-of-an-array-in-its-own-usememo-hook